src/python/CRABInterface/RESTUserWorkflow.py

```diff
@@ -511,6 +511,10 @@ def validate(self, apiobj, method, api, param, safe):
             ## used by errors and report (short format in report means we do not query DBS)
             validate_num('shortformat', param, safe, optional=True)
 
+            # used by publicationStatus
+            validate_str("asourl", param, safe, RX_ASOURL, optional=True)
+            validate_str("asodb", param, safe, RX_ASODB, optional=True)
```

Comment:
MM: decided to not do something like:

```python
if subreasource=='publicationstatus' and not (asourl and asodb):
    rainse InvalidParameter("a nice error")
```

because asourl and asodb are deprecated for oracle, and they are always be set by the client anyway